adapters/repos/db/inverted_reindexer_map_to_blockmax.go

```diff
@@ -721,15 +721,15 @@ func (t *ShardReindexTask_MapToBlockmax) getSegmentPathsToMove(bucketPathSrc, bu
 		}
 		if t.isSegmentDb(d.Name()) || t.isSegmentBloom(d.Name()) {
 			ext := filepath.Ext(d.Name())
-			id := strings.TrimSuffix(strings.TrimPrefix(d.Name(), "segment-"), ext)
-			timestamp, err := strconv.ParseInt(id, 10, 64)
+			idAndData := strings.Split(strings.TrimSuffix(strings.TrimPrefix(d.Name(), "segment-"), ext), ".")
+			timestamp, err := strconv.ParseInt(idAndData[0], 10, 64)
 			if err != nil {
 				return err
 			}
 			timestampPast := time.Unix(0, timestamp).AddDate(-23, 0, 0).UnixNano()
-
+			idAndData[0] = strconv.FormatInt(timestampPast, 10)
 			segmentPaths = append(segmentPaths, [2]string{
-				path, filepath.Join(bucketPathDst, fmt.Sprintf("segment-%d%s", timestampPast, ext)),
+				path, filepath.Join(bucketPathDst, fmt.Sprintf("segment-%s%s", strings.Join(idAndData, "."), ext)),
```

Comment:
as a follow up PR we can extract segment name parsing to dedicated methods (similarly to `segmentID()`, so if name format changes in the future, only single places would require update